pymc3/distributions/multivariate.py

```diff
@@ -1271,7 +1234,6 @@ def LKJCholeskyCov(name, eta, n, sd_dist, compute_corr=False, store_in_trace=Tru
     packed_chol = _LKJCholeskyCov(name, eta=eta, n=n, sd_dist=sd_dist)
```

Comment:
One thing I noticed is that `**kwargs` are not being passed over here, which means the distribution will always have a size of `None`. If thats the case, then I don't think we require most of the shapes logic in `rng_fn` of `LKJCholeskyCov` unless there are other use cases.